test/integration/go.mod

```diff
@@ -12,60 +12,104 @@ require (
 )
 
 require (
-	buf.build/gen/go/open-feature/flagd/connectrpc/go v1.12.0-20231031123731-ac2ec0f39838.1 // indirect
-	buf.build/gen/go/open-feature/flagd/grpc/go v1.3.0-20231031123731-ac2ec0f39838.2 // indirect
-	buf.build/gen/go/open-feature/flagd/protocolbuffers/go v1.31.0-20231031123731-ac2ec0f39838.2 // indirect
-	connectrpc.com/connect v1.14.0 // indirect
+	buf.build/gen/go/open-feature/flagd/connectrpc/go v1.15.0-20240215170432-1e611e2999cc.1 // indirect
```

Comment:
optional, but if we're doing big renames/refactors, can we perhaps also change this to `tests.e2e`? These are really e2e tests. You will also need to change the makefile and CI commands: https://github.com/open-feature/flagd/blob/517973fa84f0faaa88c12ce06285b30e655860d9/Makefile#L47-L48

Reply:
On second thought, maybe we should do this in a PR, so test changes are isolated from refactors, for increase confidence.

Reply:
Yes, i think I will defer this to another PR :) This PR only focuses on moving sources to repackage